lib/tasks/osc.rake

```diff
@@ -74,6 +74,21 @@ namespace :osc do
     end
   end
 
+  def check_IDs_in_changes_file!
+    # Checking if any new bugzilla,fate,... IDs are defined
+    # in the *.changes file
+    puts "Checking IDs in *.changes file" if verbose
+    Dir.chdir(osc_checkout_dir) do
+      cmd = "osc -A '#{obs_api}' diff *.changes | grep \"bnc#[0-9]\\|fate#[0-9]\\|boo#[0-9]\\|bsc#[0-9]\\|bgo#[0-9]\""
```

Comment:
But then the automatically submission to factory, SP1,... would be broken too, because these fixes (without changelog) would not be accepted. Or I am wrong ?